src/Control/SessionHandler/CacheSessionHandler.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace SilverStripe\Control\SessionHandler;
+
+use SensitiveParameter;
+use Psr\SimpleCache\CacheInterface;
+
+/**
+ * Session save handler that stores session data in an in a PSR-16 cache adapter.
+ */
+class CacheSessionHandler extends AbstractSessionHandler
+{
+    private CacheInterface $cacheHandler;
```

Comment:
```suggestion
    private CacheInterface $cache;
```

Note sure what the 'Handler' suffix is for? Also update all the references to it

Reply:
Oops should have been `$cacheAdapter` which is what I'll set it to.

Reply:
Where are you getting the word "adapter" from? That word is not present on https://www.php-fig.org/psr/psr-16/

`CacheInterface` implies that it's an interface for a cache?

I *think* there is a concept of an "adapter" if you want to adapt a PSR-6 cache interface to a PSR-16 cache interface, though a PSR-16 compatible cache can also just be a stand-alone cache with no compatibility to a PSR-6 cache.

ie. the PSR-16 compatible object being passed in could be a cache wrapped in an adapter, or just a cache, so $cache would be a better name as $cacheAdapter could be wrong.

Reply:
I don't think the name matters that much :sweat_smile: we both know what this variable holds. If you really really really want me to change it to `$cache` I will but I think we're just spinning wheels at this point.

> Where are you getting the word "adapter" from?

It's common parlance when dealing with symfony cache *stuff*, which is the primary cache *stuff* we tend to use.

"cache" on its own is an overloaded word. I tend to think of "cache" as the content that is _stored_ or the place _where_ it's stored.... this variable represents neither of those. It represents the API that is used to take some data and store it in the cache. Adapter seems like a good word for that. I guess `$cacheTransport` would also make sense for that but I've not seen that terminology used anywhere before.

Reply:
Change it to $cache please :-)

Reply:
Sure. Done.